code/modules/spells/spell_types/shadow_walk.dm

```diff
@@ -78,3 +82,29 @@
 			visible_message("<span class='boldwarning'>[jaunter] emerges from the darkness!</span>")
 		playsound(loc, 'sound/magic/ethereal_exit.ogg', 50, TRUE, -1)
 	qdel(src)
+
+/obj/effect/dummy/phased_mob/shadow/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/effect/dummy/phased_mob/shadow/Destroy()
+	STOP_PROCESSING(SSobj, src)
```

Comment:
Add `jaunter = null` here too.

Reply:
Is that needed? Or is it a cleanliness thing